tests/Behat/Gherkin/Node/OutlineNodeTest.php

```diff
@@ -122,11 +123,37 @@ public function testPopulatesExampleWithOutlineTitle()
         $table = new ExampleTableNode(array(
             10 => array('name', 'email'),
             11 => array('Ciaran', 'ciaran@example.com'),
+            12 => array('John', 'john@example.com'),
         ), 'Examples');
 
-        $outline = new OutlineNode('An outline title', array(), $steps, $table, null, null);
-
-        $this->assertCount(1, $examples = $outline->getExamples());
-        $this->assertEquals('An outline title', current($examples)->getOutlineTitle());
+        $outline = new OutlineNode('An outline title for <name>', array(), $steps, $table, null, null);
+
+        $this->assertEquals(
```

Comment:
I don't think it makes any difference here, but I always try to use `assertSame` unless there's a reason not to - just avoids any risk of false positives if someone adds another case (e.g. maybe with empty strings) in the future and doesn't notice the loose assertion.

```suggestion
        $this->assertSame(
```

Reply:
Agreed, that's also an old habit of mine - in the past (PHPUnit 5 or 4 maybe?) `assertSame` didn't show the level of detail (e.g. a diff) like `assertEquals` :)

Reply:
Ah yeah, I remember that...